cobbler/modules/authentication/__init__.py

```diff
@@ -1,4 +1,25 @@
 """
 This module represents all Cobbler methods of authentication. All present modules may be used through the configuration
 file ``modules.conf`` normally found at ``/etc/cobbler/``.
+
+In the following the specification of an authentication module is given:
+
+1. The name of the only public method - except the generic ``register()`` method -  must be ``authenticate``
+1. The attributes are - in exactly this order: ``api_handle``, ``username``, ``password``
+1. The username and password both must be of type ``str``.
+1. The ``api_handle`` must be the main ``CobblerAPI`` instance.
+1. The return value of the module must be a ``bool``.
+1. The method should only return ``True`` in case the authentication is successful.
+1. Errors should result in the return of ``False`` and a log message to the standard Python logger obtioned via
+   ``logging.getLogger()``.
+1. The return value of ``register()`` must be ``authn``.
```

Comment:
Shouldn't these count up?

Reply:
We were both wrong. I need to use hashtags: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/lists.html

Reply:
Fixed via force push